var/spack/repos/builtin/packages/postgresql/package.py

```diff
@@ -64,7 +64,9 @@ class Postgresql(AutotoolsPackage):
     variant("tcl", default=False, description="Enable Tcl bindings.")
     variant("gssapi", default=False, description="Build with GSSAPI functionality.")
     variant("xml", default=False, description="Build with XML support.")
+    variant("icu", default=True, description="Build with ICU support.")
```

Comment:
```suggestion
    variant("icu", default=True, description="Build with ICU support.", when="@16:")
```

Is the option to build with ICU support available before v16?

Reply:
It's not. Thanks for catching that.